solidity/contracts/relays/OpenRelay.sol

```diff
@@ -17,25 +17,56 @@ contract OpenRelay is IOpenRelay {
   /// @inheritdoc IOpenRelay
   uint32 public constant BASE = 10_000;
 
+  mapping(IAutomationVault _automationVault => mapping(address _job => PaymentData _paymentData)) public paymentsData;
+
+  function setExtraPayment(IAutomationVault _automationVault, address _job, PaymentData memory _paymentData) external {
+    if (_automationVault.owner() != msg.sender) revert OpenRelay_NotVaultOwner();
```

Comment:
isn't there any modifiers to use? `onlyVaultOwner` or sth like?

Reply:
I can create a new one, also we could create a library with some modifers because are used in several contracts. WDYT

Reply:
i'm not sure about a library, because you need the `owner` storage variable, but i'd abstract it into an inherited contract maybe, yes